tests/CallbackScheduler_test.cdc

```diff
@@ -64,10 +87,100 @@ access(all) fun testCallbackSchedulingAndExecution() {
     let scheduled = scheduledCallbacks[0]
     Test.assert(scheduled.id == callbackID, message: "callback ID mismatch")
     Test.assert(scheduled.timestamp == futureTime, message: "incorrect timestamp")
-    Test.assert(scheduled.status() == FlowCallbackScheduler.Status.Scheduled, message: "incorrect status")
 
-    var status = FlowCallbackScheduler.getStatus(id: callbackID)
-    Test.assertEqual(FlowCallbackScheduler.Status.Scheduled, status!)
+    var status = executeScript(
+        "../transactions/callbackScheduler/scripts/get_status.cdc",
+        [callbackID]
+    ).returnValue! as! UInt8
+    Test.assertEqual(UInt8(0), status!)
+
+    // Schedule another callback, medium this time
+    tx = Test.Transaction(
+        code: Test.readFile("../transactions/callbackScheduler/schedule_callback.cdc"),
+        authorizers: [serviceAccount.address],
+        signers: [serviceAccount],
+        arguments: [futureTime, feeAmount, mediumEffort, mediumPriority, testData],
+    )
```

Comment:
maybe good idea to extract this into helper function `scheduleCallback(timestamp, fee, effort, priority, data)`

Reply:
yeah, I think I'll do that